portfolio/src/main/webapp/script.js

```diff
@@ -39,66 +39,47 @@ function deleteComment(key){
     });
 }
 
-let map=null;
-function createMap() {
-    map = new google.maps.Map(
-    document.getElementById('map'),
-    {center: {lat: 37.422, lng: -122.084}, zoom: 4, mapTypeId: 'hybrid'});
-
-    let homeMarkerInfo = new google.maps.InfoWindow({
-        content: "<h6> Phoenix </h6>" + "</br>" + "<p>I have grown up in Arizona all my life. I lived in Virgina for three months when I was eight because my dad is from there but otherwise I have only lived in AZ</p>"
+function createMarkerWindow(marker, message) {
```

Comment:
even though we are using a global "map" variable in this file, we should explicitly pass the map variable to this function. We might want to use this again with another map that is not global so that change would increase the reusability of the code.